api/authorizers/site.js

```diff
@@ -51,7 +51,7 @@ const authorizeRepositoryAdmin = (user, site) => (
 );
 
 function authorizeFederalistUsersAdmin(user) {
```

Comment:
Do we still need this function (and the `UsersHelper` service) at all? The only place I see it used is when destroying a site, we check that it's a repo admin first, and then fallback to checking if it's a federalist-users org admin on error. I think we should eliminate it entirely (and potentially replace that second check with "is a Pages admin" in the future)

Reply:
Removed `UsersHelper` and the associated admin functionality in the app routes. This update will not affect any of the admin routes.